experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/HiveUtils.java

```diff
@@ -141,107 +139,92 @@ public static Storage toProto(StorageDescriptor sd, PathTranslator translator)
    */
   public static ColumnStatisticsInfo toProto(ColumnStatisticsObj colStats) {
     ColumnStatisticsInfo.Builder builder = ColumnStatisticsInfo.newBuilder();
-    FieldTypeId type = toProto(colStats.getColType());
-    builder.setColName(colStats.getColName()).setColType(type);
-    switch (type) {
-      case BOOLEAN: {
-        org.apache.hadoop.hive.metastore.api.BooleanColumnStatsData data =
-            colStats.getStatsData().getBooleanStats();
-        if (data != null) {
-          builder.setData(ColumnStatisticsData.newBuilder()
-              .setBooleanStats(BooleanColumnStatsData.newBuilder()
-                  .setNumTrues(data.getNumTrues()).setNumFalses(data.getNumFalses())
-                  .setNumNulls(data.getNumNulls()).setBitVectors(data.getBitVectors())
-                  .build()).build());
-        }
-        break;
+    builder.setColName(colStats.getColName()).setColType(colStats.getColType());
+    if (colStats.getStatsData().isSetBooleanStats()) {
```

Comment:
NIT: `colStats.getStatsData()` is used many times, can we just save it to a local variable?

Reply:
fixed